PROJ-138 - MVP (#80)
SW805F20/report: sections/introduction/MVP.tex

\section{Minimum viable product}
To be able to evaluate the game with users as fast as possible, we establish a minimal viable product with functional and non-functional requirements.

\subsection{Functional requirements}
These functional features are essential to make the game work and are the core features of the game.

\begin{itemize}
    \item Must be possible to start the game.
    \item Must be able to connect to the server from the clients.
    \item The users must be able to see their current positions.

>     \item The users must be able to view the playing field, the goals, the players, the ball position and the current score on their mobile device.

Reviewer: ```suggestion
    \item The users must be able to view the playing field, the goals, the players, the ball position, and the current score on their mobile device.
```

>     \item They must be able to win the game.

Reviewer: Du skriver the users i de andre, skal det ikke også være her? 
```suggestion
    \item A team must be able to win the game based on their score.
```

>     \item The game must be able to be viewed through VR googles.

Reviewer: ```suggestion
    \item The game must be able to be viewed through VR goggles.
```

Reviewer: Er det her ikke ret meget en del af "The users must be able to view the playing field, the goals, the players, the ball position, and the current score on their mobile device."

    \item The game must be dynamic, which includes:
          \begin{itemize}

>               \item Possibility to add an arbitrary number of players.

Reviewer: ```suggestion
              \item Support an arbitrary number of players.
```

              \item Size of the playing field can be set to a preferable size by the user.
              \item Set how many goals needed to score to win.
          \end{itemize}
\end{itemize}

\subsection{Non-functional requirements}
These requirements are more difficult to evaluate than the functional requirements and to test it a usability test must be conducted.
\begin{itemize}

>     \item The design of the field must be pleasant for the user to see through a VR headset.

Reviewer: ```suggestion
    \item The design of the field must be pleasant for the user to look at through a VR headset.
```

>     \item The users should receive enough updates, so that the positions of the players represent their real position.

Reviewer: ```suggestion
    \item The users should receive enough updates so that the positions of the players represent their real position.
```
